test/functional/readpreference.test.js

```diff
@@ -655,3 +656,77 @@ describe('ReadPreference', function() {
     });
   });
 });
+
+describe.only('ReadPreference - operations', () => {
```

Comment:
also does this need to be its own `describe` block? Seems like it could be nested within `ReadPreference` (and indeed is named as if it was). What about:
```
describe('ReadPreference', function() {
  context('write operations', function() {
    beforeEach(() => ...);
    Object.keys(..).forEach(operation => {
      it('should... ', () => ...);
    });
  });
});
```